package/MDAnalysis/core/groups.py

```diff
@@ -2689,6 +2689,10 @@ def select_atoms(self, sel, *othersel, **selgroups):
             record_type *record_type*
                 for selecting either ATOM or HETATM from PDB-like files.
                 e.g. ``select_atoms('name CA and not record_type HETATM')``
+            smarts *SMARTS-query*
```

Comment:
So with atom selections, there's two sets of documentation.  Somewhere there's a `.rst` file with essentially these docs duplicated but expanded a bit.  Can you find these and document `smarts` there? (Unless I'm wrong @lilyminium ?)

Reply:
Yeah there's `selections.rst` in `sphinx/documentation_pages`; redundantly, there's also a separate copy in the user guide :/ we should pick which one we like and point to it instead of having 3 versions. I think the user guide *might* be the most up-to-date one at this point?